utils/data.py

```
@@ -0,0 +1,108 @@
+
+def add_item(filename, item, id_field="id"):
+    """새 항목 추가"""
+    data = load_json(filename)
+
+    # ID 자동 생성
+    if id_field not in item:
+        item[id_field] = get_next_id(data, id_field)
+
+    # 생성 시간 추가
+    if "createdAt" not in item:
+        item["createdAt"] = datetime.now().isoformat()
```

Ruff (0.14.11)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)